notebooks/Annotation and Continuous Learning Overview.ipynb

```diff
@@ -64,11 +64,11 @@
    "source": [
     "# API Endpoint and Credentials\n",
     "host_url = \"https://api.monai.ngc.nvidia.com\"\n",
-    "ngc_api_key = os.environ.get('MONAI_API_KEY')\n",
+    "ngc_api_key = os.environ.get(\"MONAI_API_KEY\", \"<YOUR_API_KEY>\")  # we recommend using environment variables for API keys, but you can also hardcode them here\n",
     "# Dicom Server\n",
-    "dicom_web_endpoint = \"<DICOMWeb address>\" # For example \"http://127.0.0.1:8042/dicom-web\".\n",
-    "dicom_client_id = \"<DICOMWeb user ID>\"    # If Authentication is enabled, then provide username\n",
-    "dicom_client_secret = \"<DICOMWeb secret>\" # If Authentication is enabled, then provide password"
+    "dicom_web_endpoint = \"<DICOMWeb address>\" # Please fill it with the actually endpoint (usually ended with /dicom-web). For example \"http://127.0.0.1:8042/dicom-web\".\n",
+    "dicom_client_id = \"<DICOMWeb user ID>\"    # If Authentication is enabled, then provide username, otherwise fill it with the default username \"orthanc\"\n",
+    "dicom_client_secret = \"<DICOMWeb secret>\" # If Authentication is enabled, then provide password, otherwise fill it with the default password \"orthanc\""
```

Comment:
Does this apply to other DICOMWeb servers as well? E.g. DCM4CHEE? No changes needed but we should consider the ones we validate again

Reply:
I will create separate issue for this.

Reply:
Issue in https://github.com/NVIDIA/monai-cloud-api/issues/80